crates/storage-pg/src/user/registration.rs

```diff
@@ -851,4 +900,120 @@ mod tests {
             .await;
         assert!(res.is_err());
     }
+
+    #[sqlx::test(migrator = "crate::MIGRATOR")]
+    async fn test_set_upstream_oauth_link(pool: PgPool) {
```

Comment:
maybe `_link` is a slight misnomer / not consistent with the other methods

Reply:
That's a leftover for when it was the link that was attached to registrations, not the session! Fixed in 61ee8dae8